packages/core/src/common/classes.ts

```diff
@@ -116,6 +144,11 @@ export const TAB_LIST = "pt-tab-list";
 export const TAB_PANEL = "pt-tab-panel";
 export const TABS = "pt-tabs";
 
+export const TABLE = "pt-table";
+export const CONDENSED = "pt-condensed";
+export const STRIPED = "pt-striped";
+export const BORDERED = "pt-bordered";
```

Comment:
interesting quirk: these last three are technically modifiers but they're only supported by this one component. 

I wonder if we should call the constants `TABLE_*` to indicate that's where they're intended to be used? 

And then if we should change the classes to `pt-table-*` in 2.x?